screens/LIneItemEdit.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react'
+import {
+  View,
+  Text,
+  ScrollView,
+  AsyncStorage,
+  ActivityIndicator
+} from 'react-native'
+import { connect } from 'react-redux'
+import LIneItemForm from './LIneItemForm'
+import { getOrder } from '../actions'
+import { api, makeFetchRequest } from '../constants/Backend'
+import styles from '../styles'
+
+class LIneItemEdit extends Component {
+  static navigationOptions = {
+    header: null
+  }
+
+  componentDidMount() {
+    this.props.getOrder()
+  }
+
+  // handleEditCancel = () => {
```

Comment:
can remove commented out code.

Reply:
every commented code is removed for each review.